### PR TITLE
Improve UI

### PR DESCRIPTION
**Pull request by helfi92**
Screenshot_2020-10-02 Taskcluster

### CODE REVIEW

**Review by imbstack — APPROVED** [review]
nice!

File: src/Home.jsx [hunk 1]
```diff
@@ -24,307 +26,538 @@ import CogBoxIcon from 'mdi-react/CogBoxIcon';
...
+              <Typography variant="subtitle1">
+                Taskcluster is the task execution framework that supports
+                Mozilla&#39;s continuous integration and release processes. It
+                is a flexible, scalable open-source framework that can be used
+                to build complex, highly customized CI systems.
+              </Typography>
+              <div className={classes.docBtnDiv}>
+                <Link
+                  target="_blank"
+                  href="https://docs.taskcluster.net/docs/manual/using/integration">
```
Comment (djmitche) [inline src/Home.jsx:367]: Why not just https://docs.taskcluster.net?

File: src/Home.jsx [hunk 2]
```diff
@@ -24,307 +26,538 @@ import CogBoxIcon from 'mdi-react/CogBoxIcon';
...
+              There are lots of continuous integration (CI) solutions out there
+              that support simple workflows, but what if your CI process is more
+              complex? What if you need something more customizable and more
+              scalable?
+            </Typography>
+            <Typography
+              variant="h5"
+              className={classes.enterTaskcluster}
+              gutterBottom>
+              ENTER TASKCLUSTER.
```
Comment (djmitche) [inline src/Home.jsx:422]: I love this callout

**Review by djmitche — APPROVED** [review]
Thank you for interpreting my childish scrawls into something beautiful :)

**Pull request merged**